outlines/models/openai.py

```diff
@@ -439,7 +439,11 @@ def openai_model(
         config = OpenAIConfig(model=model_name)
 
     client = AsyncOpenAI(**openai_client_params)
-    tokenizer = tiktoken.encoding_for_model(model_name)
+
+    try:
+        tokenizer = tiktoken.encoding_for_model(model_name)
+    except KeyError:
```

Comment:
raise a warning so the user knows we made a choice for them, and what this choice is